library/blob.c

```diff
@@ -2037,11 +2067,12 @@ int eblob_write_commit(struct eblob_backend *b, struct eblob_key *key,
 		wc.flags |= BLOB_DISK_CTL_NOCSUM;
 
 	err = eblob_write_commit_nolock(b, key, &wc);
```

Comment:
Calling function named *_nolock() not under the lock looks suspicious, why is it safe?